agent/ec2/ec2_metadata_client.go

```diff
@@ -16,33 +16,25 @@ package ec2
 import (
 	"encoding/json"
 	"errors"
-	"fmt"
-	"io/ioutil"
-	"net"
-	"net/http"
 	"strings"
 	"time"
 
-	"github.com/aws/amazon-ecs-agent/agent/utils"
-	"github.com/cihub/seelog"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/ec2metadata"
+	"github.com/aws/aws-sdk-go/aws/session"
 )
 
 const (
-	EC2_METADATA_SERVICE_URL                      = "http://169.254.169.254"
-	SECURITY_CREDENTIALS_RESOURCE                 = "/2014-02-25/meta-data/iam/security-credentials/"
-	INSTANCE_IDENTITY_DOCUMENT_RESOURCE           = "/2014-02-25/dynamic/instance-identity/document"
-	INSTANCE_IDENTITY_DOCUMENT_SIGNATURE_RESOURCE = "/2014-02-25/dynamic/instance-identity/signature"
-	SIGNED_INSTANCE_IDENTITY_DOCUMENT_RESOURCE    = "/2014-02-25/dynamic/instance-identity/pkcs7"
-	EC2_METADATA_REQUEST_TIMEOUT                  = time.Duration(1 * time.Second)
+	SECURITY_CREDENTIALS_RESOURCE                 = "iam/security-credentials/"
```

Comment:
Since you're here...can you make these `GoStyleConsts` instead of `C_STYLE_CONSTS`?